src/libs/OptionsListUtils.js

```diff
@@ -22,6 +22,18 @@ Onyx.connect({
     callback: val => currentUserLogin = val && val.email,
 });
 
+let currentlyViewedReportID;
+Onyx.connect({
+    key: ONYXKEYS.CURRENTLY_VIEWED_REPORTID,
+    callback: val => currentlyViewedReportID = val,
+});
```

Comment:
NAB & FYI but I figured out a way to get rid of `currentlyViewedReportID` completely by upgrading some `react-navigation` stuff and am able to pass the reportID from the route and use it as a single source of truth. Not sure if that will affects your changes much but thought you would be interested in that.